sdk/cosmosdb/cosmos/src/client/ChangeFeed/ChangeFeedForEpkRange.ts

```diff
@@ -406,6 +406,13 @@ export class ChangeFeedForEpkRange<T> implements ChangeFeedPullModelIterator<T>
       feedOptions.initialHeaders[Constants.HttpHeaders.IfModifiedSince] = this.startTime;
     }
     const rangeId = await this.getPartitionRangeId(feedRange, diagnosticNode);
+    if (this.clientContext.enableEncryption) {
```

Comment:
can we extract this to a helper function? It is repetitive across multiple files

Reply:
if (!this.container._rid) {
        const { resource: containerDefinition } = await this.container.read();
        this.container._rid = containerDefinition._rid;
}
this part